src/uu/ln/src/ln.rs

```diff
@@ -338,10 +341,20 @@ fn link_files_in_dir(files: &[PathBuf], target_dir: &Path, settings: &Settings)
                 }
             };
 
-        if let Err(e) = link(srcpath, &targetpath, settings) {
+        if linked_destinations.contains(&targetpath) {
+            // If the target file was already created in this linked call, do not overwrite
+            show_error!(
+                "will not overwrite just-created '{}' with '{}'",
```

Comment:
I guess this error message is different from GNU's for consistency reasons with `cp` and `mv`?

Reply:
yeah